cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -660,6 +667,30 @@ def _select_ds_for_volume(self, volume, host=None, create_params=None):
 
         return (host_ref, resource_pool, folder, summary)
 
+    @property
+    def service_locator_info(self):
+        """Returns information needed to build a ServiceLocator spec."""
+        # vCenter URL
+        host = self.configuration.vmware_host_ip
+        port = self.configuration.vmware_host_port
+        url = "https://" + host
+        if port:
+            url += ":" + str(port)
+        # ssl thumbprint
+        cert = ssl.get_server_certificate((host, port or 443))
+        x509 = OpenSSL.crypto.load_certificate(OpenSSL.crypto.FILETYPE_PEM,
```

Comment:
This could be done by the local driver, too, and brings no security benefit imho. Is there no way to ask the vCenter with our current connection for a certificate thumbprint? @imitevm Do you have any idea?

Reply:
Maybe we can skip this.
I see that thumbprint is not a mandatory parameter.

Reply:
We also have to check if it's ok for the vCenter to connect to invalid SSL certificates. We're able to specify `verify=False` in `oslo.vmware` iirc, but how about the vCenter?